interface/patient_file/summary/dashboard_react/app/_devapp/components/MedicalHighlight.js

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+
+import Table from 'react-bootstrap/Table';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import Collapse from 'react-bootstrap/Collapse';
+import helpers from '../utils/helpers.js';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import ToggleButton from "./global/ToggleButton";
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEdit } from '@fortawesome/free-solid-svg-icons'
+library.add( faEdit )
+
+
+
+class MedicalHighlight extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            patientId: props.patientId,
+            data: [],
+            element:props.element,
+            element_title:props.element_title,
+            renderElement:false,
+            isOpen:false
+        }
+    }
+    setElement(  element,element_title){
+        this.setState.element=element;
+        this.setState.element_title=element_title;
+    }
+    getMedicalProblems() {
+        this.setState.patientId = helpers.getPatientId();
+        if (this.setState.patientId >= 0) {
+
+            fetch("../../../../apis/api/patient/" + this.setState.patientId+"/"+this.returnUrl(this.state.element), {
```

Comment:
The base URL for the API should be extracted into a global constant.  If the api base path changes then it only has to be taken care of in one place is my view.